osquery/tables/yara/yara.cpp

```diff
@@ -39,18 +40,57 @@ FLAG(bool,
      true,
      "Call malloc_trim() after YARA scans (linux)");
 #endif
+
 FLAG(uint32,
      yara_delay,
      50,
      "Time in ms to sleep after scan of each file (default 50) to reduce "
      "memory spikes");
 
+FLAG(bool,
+     enable_yara_table_extension,
```

Comment:
In osquery parlance, this is not an extension. Maybe `enable_yara_sigrule`?

Though probably should also gain `enable_yara_string_return`

Reply:
I updated the configuration flag to `enable_yara_sigrule` and the rule strings are set to private by default. The private strings does not show-up in the yara table. `disable_yara_string_private` flag can be used to disable making strings to private.